### ISSUE TITLE
MySQL Issues

### ISSUE THREAD

**Issue by aewing** [facebookarchive/heroku-buildpack-hhvm#20]
I'm getting the following error when trying to use this package with Ubuntu 14.04 x64:

remote: vendor/hhvm/hhvm: error while loading shared libraries: libmysqlclient_r.so.16: cannot open shared object file: No such file or directory

**Comment by Mihailoff** [comment]
Any news how to solve this?

**Comment by bountin** [comment]
+1 (and I don't even need MySQL)

**Comment by peterkinnaird** [comment]
+1 (also dont need mysql)

**Comment by henrikbjorn** [comment]
I think https://github.com/heroku/heroku-buildpack-php should be used instead of this buildpack.

**Comment by bountin** [comment]
Oh, I wasn't aware of that - I just moved and it works :)

Maybe @CHH or @ptarjan could put this information into the readme?

**Comment by JoelMarcey** [comment]
@bountin Or you can always send a pull request for the README too :)

**Comment by bountin** [comment]
@JoelMarcey What I can't tell is if there are plans to continue support for this build pack or to deprecate it in favor of the official one. Looking at the open issues and the last commit I think it is the latter one but I think it's rude for an outsider to suggest something like this in a pull request. But if it's okay with you I'll provide one.

**Comment by JoelMarcey** [comment]
@jwatzman Do you know whether or not we are deprecating the Heroku build pack?

**Comment by jwatzman** [comment]
Yeah I'm not sure what the status of this is -- I _think_ this was a proof-of-concept that Heroku turned into something official? Not 100% sure. @ptarjan do you know?

**Comment by ptarjan** [comment]
Yes, exactly. You should use the hhvm support by heroku: https://devcenter.heroku.com/articles/php-support
